src/components/Topology/Property/property.js

```diff
@@ -1,13 +1,30 @@
-import React from "react";
+import React, { useMemo } from "react";
 import PropTypes from "prop-types";
 import cx from "clsx";
-import { Icon } from "../../Icon";
+import { BiGlobe, BiMapAlt, BsClock, IoHardwareChip } from "react-icons/all";
 
 export const Property = ({ property, className }) => {
   const { name, text, icon, color } = property;
+  const PropertyIconName = {
+    Name: "name",
+    Account: "account",
+    Region: "region",
+    Dns: "dns"
+  };
+  const choosePropertyIcon = {
```

Comment:
The properties cannot be hardcoded - they are dynamic and come from the end user who can specify anything in src/icons

Reply:
reverted